articles/server-platforms/rails/00-introduction.md

```diff
@@ -0,0 +1,78 @@
+## Introduction
+This is the very beginning of a simple, practical and multi-step quickstart that will guide you through managing authentication in your Ruby on Rails apps with Auth0.
+
+### Seed & Samples
+
+The Auth0 RoR samples provide a seed project and a set of samples.
+- The seed is a regular RoR app with nothing on Auth0 in it except for the dependencies listed in this article and your Auth0 API keys. It serves as the starting point for building an app that works with Auth0 from scratch.
+- There is a sample for each step on this quickstart. The samples are nothing but the seed plus the contents specified on each step, allowing you to understand how to implement a particular functionality related to Auth0.
+
```

Comment:
Please replace this text (lines 6-8) with the following (if you think anything is wrong please edit it):

```
There are two options to following along this quickstart. You can either download the [seed project](https://github.com/auth0-samples/auth0-rubyonrails-sample/tree/master/00-Starter-Seed) or the [samples](https://github.com/auth0-samples/auth0-rubyonrails-sample) provided at each page of this quickstart.

The seed is a regular RoR app, with all the Auth0 dependencies set, but nothing more. It's an empty canvas meant to be filled as you follow along the steps of this quickstart. If you prefer this option download the seed from our [GitHub repository](https://github.com/auth0-samples/auth0-rubyonrails-sample/tree/master/00-Starter-Seed) and follow along.

Instead you can choose to follow the samples that are included in each step. Each sample uses the seed project as a starting point and applies to it the configuration of each step, so for example the Login sample would be the seed project plus the configuration required to implement login functionality. If you choose to follow this approach continue reading, the rest of this document will guide you through setting up the required prerequisites.
```